test/builder/QuarkBuilderCometSupply.t.sol

```diff
@@ -444,6 +533,141 @@ contract QuarkBuilderCometSupplyTest is Test, QuarkBuilderTest {
         assertNotEq(result.eip712Data.hashStruct, hex"", "non-empty hashStruct");
     }
 
+    function testCometSupplyMaxWithBridge() public {
+        QuarkBuilder builder = new QuarkBuilder();
+        QuarkBuilder.BuilderResult memory result = builder.cometSupply(
+            cometSupply_(8453, type(uint256).max),
+            chainAccountsList_(6e6), // holding 3 USDC in total across chains 1, 8453
+            paymentUsd_()
```

Comment:
Can you test with token payment instead? To make sure that Quotecall is being used here

Reply:
No problem 👍 
I have create anotherone to test with token payment.